modules/spring-config/src/main/java/org/mule/config/spring/parsers/generic/TextDefinitionParser.java

```diff
@@ -27,16 +27,36 @@
  */
 public class TextDefinitionParser extends ChildDefinitionParser
```

Comment:
This is just a rollback of a change that is no longer necessary with these parser changes.